src/nni_manager/training_service/dlts/dltsJobConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { DLTSClusterConfig } from "./dltsClusterConfig";
```

Comment:
Would you please add license at file beginning and a empty line at end?

Reply:
Done